globus_automate_client/cli/flows.py

```diff
@@ -132,33 +181,65 @@ def flow_deploy(
         "--keyword",
         help="A keyword which may categorize or help discover the Flow. [repeatable]",
     ),
+    flow_viewer: List[str] = typer.Option(
+        None,
+        help=(
+            "A principal which may view this Flow. "
+            + _principal_description
+            + " The special value of 'public' may be used to "
+            "indicate that any user can view this Flow. [repeatable]"
+        ),
+        callback=principal_or_public_validator,
+        hidden=False,
+    ),
+    # viewer and visible_to are aliases for the full flow_viewer
+    viewer: List[str] = typer.Option(
+        None,
+        callback=principal_or_public_validator,
+        hidden=True,
+    ),
     visible_to: List[str] = typer.Option(
         None,
-        help="A principal which may view this Flow. The principal value is the user's "
-        "or group's UUID prefixed with either 'urn:globus:groups:id:' or "
-        "'urn:globus:auth:identity:'. The special value of 'public' may be used to "
-        "indicate that any user can view this Flow. [repeatable]",
         callback=principal_or_public_validator,
+        hidden=True,
     ),
-    administered_by: List[str] = typer.Option(
+    flow_starter: List[str] = typer.Option(
```

Comment:
I don't think we need to worry about so many aliases on the CLI. I would be ok just keeping the starter, administrator, owner etc. CLI options for the sake of keeping the interface simpler, esp on this minor release update.